esy-lib/Path.re

```diff
@@ -103,9 +103,9 @@ let safeSeg = {
     |> String.lowercase_ascii
     |> Str.global_replace(replaceAt, "")
     |> Str.global_replace(replaceUnderscore, "__")
-    |> Str.global_replace(replaceSlash, "__slash__")
```

Comment:
@andreypopp - are there any concerns for making this change in terms of backwards-compatibility with existing caches?

Reply:
I guess it would mean re-fetching and re-building packages 😦

Reply:
Yeah, it will invalidate caches but let's do that, current names are too long.

Reply:
Thanks @andreypopp , sounds good!